src/tools/jsondocck/src/main.rs

```diff
@@ -65,6 +65,11 @@ enum CommandKind {
     /// Checks the path doesn't exist.
     HasNotPath,
 
+    /// `//@ !has <path> <value>`
+    ///
+    /// Checks the path doesn't exist.
```

Comment:
Isn't it supposed to check that the path exists but doesn't have this value?

Reply:
D'oh! Copy & Paste error. Fixed.

Also, I like your idea of checking the path does exist. I've added that.